packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -66,6 +66,12 @@ Will double processing times. Use only for debugging purposes.",
     group: "chain",
   },
 
+  "chain.maxSkipSlots": {
```

Comment:
@dapplion should this option to be hidden? Also should it have a default value? I do not see a default value set in LH implementation though. If there is a need to set a default, what range of value will be advisable?

Reply:
Lighthouse doesn't have a default value because it's disabled by default. If the CLI value is not set the check does not apply

Reply:
Yeah. I was asking, should we also implement it that way (that is have it disabled by default, and hence do not have a default value)? Or do we consider the possibility of a DoS via this avenue high enough that we want to have a default value? (and hence have the check enabled by default)

Reply:
With this enabled by default you can also reject valid blocks so it's a dangerous feature if miss-used. I imagine that's why Lighthouse has disabled it by default